### PR TITLE
By default file on line 5999 is renamed, and then referenced at 6475

### PR DESCRIPTION
**Pull request by wilcas**
In `fusioncatcher.py` the default value of  when `temp_flag` is set to the default value of `"yes"`, reads_filtered_transcriptome_sorted-read.map is renamed to reads_filtered_transcriptome_sorted-read_end.map.
A join command called on line 6475 tries to join reads_filtered_transcriptome_sorted-read_end.map to another file, despite this rewrite. I changed the filename to what it would be on the system (e.g. reads_filtered_transcriptome_sorted-read_end.map) this way the right file should be referenced whether or not the temp flag is set to rename or create a soft-link to the original reads_filtered_transcriptome_sorted-read.map

### CODE REVIEW

File: bin/fusioncatcher.py [hunk 1]
```diff
@@ -6472,7 +6472,7 @@ def is_known_extension(something):
         job.add('-2','1',kind='parameter')
         job.add('-t',"'\t'",kind='parameter')
         job.add('',outdir('original_important.txt'),kind='input',temp_path=temp_flag)
-        job.add('',outdir('reads_filtered_transcriptome_sorted-read.map'),kind='input',temp_path=temp_flag)
+        job.add('',outdir('reads_filtered_transcriptome_sorted-read_end.map'),kind='input',temp_path=temp_flag)
```
Comment (wilcas) [inline bin/fusioncatcher.py]: This is the change, sorry about the newlines

**Comment by ndaniel** [comment]
@wilcas 
Very good point!

**Comment by ndaniel** [comment]
This was backported to v0.99.7c and re-re-released v0.99.7c today on https://sourceforge.net/projects/fusioncatcher/files/

**Comment by wilcas** [comment]
Thanks for the prompt fix. I've marked down what appears to be an issue with file tracking with the *exon-exon* python scripts. Thanks for the prompt fix

**Pull request merged**